src/python/pants/option/global_options.py

```diff
@@ -1315,7 +1315,6 @@ def register_bootstrap_options(cls, register):
                 "`name=` anywhere.\n"
                 "  3. Remove `use_deprecated_python_macros = true` from `[GLOBAL]` in "
                 "pants.toml.\n\n"
-                "You can ignore this warning with `[GLOBAL].ignore_warnings`."
```

Comment:
This wasn't worth its own PR, so I snuck it in here. This is not true in this specific case unfortunately - this warning is triggered at init time, before logging is diverted to the engine, so ignore_warnings isn't applied to it.